### DISCUSSION TITLE
Welcome to hardinfo2 Discussions!

### DISCUSSION THREAD

**Discussion by hwspeedy** [hardinfo2/hardinfo2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by georgegabriel419** [comment]
I'm Gabriel from Nigeria. I just started my tech career. I want to help with the graphic design or translation.

**Reply by hwspeedy** [reply]
Welcome,

Have a look around and submit ideas for graphical improvements - that could be for images in the program or on the webpage or a new design layout idea.

For translations see https://github.com/hardinfo2/hardinfo2/blob/master/po/README.md

https://docs.github.com/en
The above link is is a good place to learn about git and github
NOTE: git is the version control software used by FOSS (Free and Open Source Software).
